CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # App Center SDK for Android Change Log
 
+## Version 1.9.0 (Under active development)
+
+### AppCenter
+
+* **[Security]** To enforce TLS 1.2 on all HTTPS connections the SDK makes, we are dropping support for API level 15 (which supports only TLS 1.0), the minimum SDK version thus becomes 16. Previous versions of the SDK were already using TLS 1.2 on API level 16+.
```

Comment:
maybe add `**[Breaking]**` tag in the notes for clarity?

Reply:
According it's more like a security bug. Description has the details.